doc/user/bgp.rst

```diff
@@ -1128,6 +1128,13 @@ Route Aggregation-IPv4 Address Family
 
    This command specifies an aggregate address.
 
+   In order to advertise aggregated prefix, a more specific (longer) prefix
```

Comment:
In order to advertise an aggregated prefix... ?  <note the addition of `an`>

Reply:
Yeah, sorry, my london skills...

Reply:
Fixed.